src/backend/clearboard/main.py

```diff
@@ -1,10 +1,28 @@
-"""FastAPI main module for the Clearboard application."""
-from fastapi import FastAPI
+"""FastAPI main module for the Clearboard application.
+coreAddress : string, defines the part of the url that wont change between several jitsi-box
+origins : string[], url to whitelist and on which the fastapi server should listen (basicly the core address)
+"""
+from fastapi import FastAPI, Request
+from fastapi.middleware.cors import CORSMiddleware
+
 
 app = FastAPI()
+coreAddress = "http://localhost:3000/"
```

Comment:
You should make it configurable through env variables. See fastapi documentation for best practice on configuration